ClientApp/src/app/flight-search-results/flight-search-results.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, OnInit } from '@angular/core';
+
+import {Observable, of, Subject} from 'rxjs';
+import {
+  debounceTime, distinctUntilChanged, switchMap
+} from 'rxjs/operators';
+
+import { Flight } from '../Flight';
+import { FlightService } from '../flight.service';
+import { SharedService } from '../Shared.service';
+import {FlightForSearch} from '../FlightForSearch';
+import {removeSummaryDuplicates} from '@angular/compiler';
```

Comment:
This import should be along with other angular imports. Also do you even need it?